src/views/search_suggestions/search_suggestions_index_view.js

```diff
@@ -7,15 +7,15 @@ import searchSuggestionsAdapter from '../../adapters/search_suggestions_adapter'
 export default BaseView.extend({
   template: SearchSuggestionsIndexTemplate,
 
-  initialize () {
+  initialize() {
     this.adapter = searchSuggestionsAdapter;
     this.listenTo(this.adapter.combinedSuggestions, 'reset', this.render);
   },
 
-  afterRender () {
+  afterRender() {
     this.removeSubviews();
     console.log('adapter', this.adapter);
```

Comment:
Not sure if we want to kill this line. I think it's currently the only `warning` we get from ESLint:

```
/Users/pdehaan/dev/github/universal-search-content/src/views/search_suggestions/search_suggestions_index_view.js
  17:5  warning  Unexpected console statement  no-console
```